blocks/subscriptions-block/components/SubscriptionDialog/index.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import SubscriptionDialog from '.';
+
+describe('SubscriptionDialog', () => {
+  it('renders with minimal required properties', () => {
+    const wrapper = mount(
+      <SubscriptionDialog
+        linkText="Log in"
+        linkUrl="/"
+      />,
+    );
+
+    expect(wrapper.find('a').at(0).text()).toEqual('Log in');
+    expect(wrapper.find('a').at(0).prop('href')).toEqual('/');
+  });
+
+  it('renders the button as a link', () => {
+    const wrapper = mount(
+      <SubscriptionDialog
+        linkText="Log in"
+        linkUrl="/"
+        actionText="Press Me!"
```

Comment:
There should be a test that account for only one of the action props being passed to ensure we are testing the `&&` logic in the component

Reply:
adding